homeassistant/components/husqvarna_automower/strings.json

```diff
@@ -37,9 +37,15 @@
         "name": "Returning to dock"
       }
     },
-    "switch": {
-      "enable_schedule": {
-        "name": "Enable schedule"
+    "select": {
+      "headlight_mode": {
+        "name": "Headlight mode",
+        "state": {
+          "always_on": "Always on",
+          "always_off": "Always Off",
+          "evening_only": "Evening only",
+          "evening_and_night": "Evening and night"
```

Comment:
General note: you're not saying if the lights are on or off during these moments. It can be read implicitly.

Reply:
```
"evening_only": "Only on in the evening",
"evening_and_night": "Only on in the evening and night"
```
That's a little bit long as text...